Sara/mkt 337 document rule board third column (#514)
returntocorp/semgrep-docs: docs/semgrep-app/rule-board.md

---
slug: rule-board
append_help_link: true
description: "The Rule Board is a visual representation of the rules that Semgrep App uses to scan code. Rules are cards, and are grouped into columns representing the actions undertaken (whether to block, comment, or silently monitor) when a finding surfaces."
---

import MoreHelp from "/src/components/MoreHelp"

# Rule board

> The Rule Board is a visual representation of the rules that Semgrep App uses to scan code. Rules can be organized into rulesets. Rulesets are rules related through programming language, OWASP category, framework, and so on.

Reviewer: ```suggestion
The Rule Board is a visual representation of the rules that Semgrep App uses to scan code. Rules can be organized in rulesets. Rulesets are rules related through a programming language, OWASP category, or framework.
```

What more can we say instead of "and so on" at the end? We can avoid it as well. Search for "etc." in https://developers.google.com/style/word-list?hl=en

Rules and rulesets are displayed as **cards**. Group cards by dragging and dropping cards into the columns. Columns represent the actions undertaken in response to findings from that rule or ruleset.

The columns and their corresonding actions are:

<dl>
    <dt>Audit</dt>

>     <dd>Rules here show findings only on Semgrep App.</dd>

Reviewer: ```suggestion
    <dd>Rules that display findings only on Semgrep App.</dd>
```

    <dt>PR/MR Comments</dt>

>     <dd>Rules here show findings to developers through PR or MRs.</dd>

Reviewer: ```suggestion
    <dd>Rules that display findings to developers through PR or MRs.</dd>
```

    <dt>Block</dt>

>     <dd>Rules here show block merges and commits, in addition to showing findings in Semgrep App and PRs or MRs.</dd>

Reviewer: ```suggestion
    <dd>Rules that block merges and commits, in addition to showing findings in Semgrep App and PRs or MRs.</dd>
```
